### ISSUE TITLE
Qube firewall traffic blocked when resolving hostname in "Firewall rules" fails

### ISSUE THREAD

**Issue by MichaelAnders** [QubesOS/qubes-issues#7499]
### Qubes OS release
Qubes 4.1

### Brief summary
If one hostname which is defined under "Firewall Rules" fails to resolve, the network is not accessible. All network traffic is blocked, even when using an IP. The packets are filtered by (according to "ping" output) the "Net qube".
It does not matter if "Allow all outgoing connections", "Limit outgoing connections to..." or "Allow full access for ,,," is selected.

### Steps to reproduce
At least one hostname is specified as "Address" under "Firewall Rules" and such a hostname cannot be resolved

### Expected behavior
Upon launching the Qube a notification is displayed for each hostname that cannot be resolved (alternative: "multiple hostnames cannot be resolved" instead of single hostname).
The network remains accessible, the failed hostname obviously not.

### Actual behavior
Upon launching the Qube a notification is displayed for the first hostname that cannot be resolved. This notification will show only once even if multiple hostnames fail.
It does not matter if "Allow all outgoing connections", "Limit outgoing connections to..." or "Allow full access for ..." is selected: all network connections are blocked.

**Comment by 3hhh** [comment]
No, the current behaviour is fine.

Assume a set of rules such as
deny somehost
allow all

If the Qubes firewall just continued when it has issues with `somehost`, it would inevitably lead to a potential security impact for the user (who might not even notice the issue). So failing closed is in the best interest of the user atm.

Whether the error message could be improved or not is the only point worth discussing.

**Comment by 3hhh** [comment]
Not necessarily. It can still mean an issue on layer 7, e.g. if the hostname of your cloud IPS, logging, ... solution could not be resolved and thus all the additional security stuff that you put in place doesn't work.

I guess a question popup whether the offending rule entry can be removed or whether to retry resolving would be more safe.
Or just open up the firewall panel and show the offending rules in red.
